tests/Resources/Page/ODH/JupyterHub/JupyterHubSpawner.robot

```diff
@@ -365,3 +366,9 @@ Verify Image Can Be Spawned
     Spawn Notebook With Arguments    ${retries}    ${image}    ${size}
     ...    ${spawner_timeout}    ${gpus}    ${refresh}    &{envs}
     End Web Test
+
+Verify Library Version Is Greater Than
+    [Arguments]     ${library}      ${target}
+    ${ver} =  Run Cell And Get Output   !pip show ${library} | grep Version: | awk '{split($0,a); print a[2]}' | awk '{split($0,b,"."); printf "%s.%s.%s", b[1], b[2], b[3]}'
+    ${comparison} =  GTE  ${ver}  ${target}
+    [Return]  ${comparison}
```

Comment:
Why are you returning the Status ? I don't see you are using it anywhere

Reply:
If you are talking about the comparison status then the task is all about that only. I have to compare with the expected version.

Reply:
I know it about comparison why are you not returning the comparison if you are not using it in your test case? What will happens if version is not greater than, it should FAIL. Could you verify with wrong version and sen me the output via chat for negative test case?